bitnami/wavefront-adapter-for-istio/values.yaml

```diff
@@ -356,10 +376,12 @@ istio:
   ## @param istio.apiVersion Override Istio API version
   ##
   apiVersion: ""
-
 ## Adapter Metrics parameters
 ##
 metrics:
+  ## @param metrics.enabled Whether or not to create a standalone Adapter exporter to expose its metrics
+  ##
+  enabled: false
```

Comment:
In this case, `metrics` parameters are used if `istio.create` is enabled. So there is no point in adding this parameter.